spec.md

```diff
@@ -1172,38 +1172,62 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 
 A Controller Plugin MUST implement this RPC call.
 This RPC will be called by the CO to check if a pre-provisioned volume has all the capabilities that the CO wants.
-This RPC call SHALL return `supported` only if all the volume capabilities specified in the request are supported.
+This RPC call SHALL return `confirmed` only if all the volume capabilities specified in the request are supported.
```

Comment:
This wording still isn't quite right. `confirmed` may contain a subset of what was validated